adjust tutorial android (#600)
ZupIT/beagle-docs: content/en/get-started/installing-beagle/android.md

## **Step 2: Include plugins**

> Once you made the first configuration, now you have to include the kapt and Beagle's plugin as dependencies on your dependency manager, below is an example.

Reviewer: ```suggestion
Once you made the first configuration, now you have to include the `kapt` and Beagle's plugin as dependencies on your dependency manager, as in the example below.
```
